### ISSUE TITLE
Overzichtspagina: card content overflow bij Firefox zoom text only

### ISSUE THREAD

**Issue by TessaViergever** [nl-design-system/mijn-services#310]
Bij Firefox zoom text only komt de content buiten de card. (verschilt per thema meer of minder).
⚠️ mogelijk een violation.

<img width="635" height="272" alt="Image" src="https://github.com/user-attachments/assets/59833431-9ff7-4328-b3b8-d465da40ebd8" />
